components/NotificationPopup/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+
+import { useGetAllNotification } from "@/hooks/useGetAllNotification";
+
+import styles from "./NotificationPopup.module.scss";
+import BaseNotification from "../Notification/BaseNotification";
+import EmptyNotification from "../Notification/EmptyNotification";
+
+const cn = classNames.bind(styles);
+
+export default function NotificationPopup() {
+  const [isClicked, setIsClicked] = useState(false);
+
+  const { data: notificationInfo } = useGetAllNotification();
+
+  const getNotificationCount = () => {
+    if (!notificationInfo) return;
+
+    return notificationInfo.count < 10 ? notificationInfo.count : "+";
```

Comment:
```suggestion
    return count < 10 ? count : "9+";
```
P4: 이렇게 하면 카운트가 10이 넘으면 `+`만 나오지 않나요??

Reply:
헉... 새벽에 코드 짰더니... 정신줄을 놓고 만들었나봐여... 수정하겠습니다ㅠㅠ